src/ui/views/Dashboard/index.tsx

```diff
@@ -84,6 +87,16 @@ const GnosisAdminItem = ({
 };
 
 const Dashboard = () => {
+  const { currentAccount, alianName, pendingTxCount } = useSelector(
+    (state: RabbyRootState) => ({
```

Comment:
prefer creating a pre-typed  hook with the correct type of state built-in instead.